qa/workunits/fs/misc/dac_override.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/sh -x
+
+set -e
+sudo mkdir -p testdir
+
+file=test_chmod.$$
+
+sudo echo "foo" > testdir/${file}
```

Comment:
wouldn't this fail if you're assuming the script is not run with root privileges (which is why you're using "sudo")?
https://stackoverflow.com/questions/82256/how-do-i-use-sudo-to-redirect-output-to-a-location-i-dont-have-permission-to-wr
Maybe, 
```
echo "foo" | sudo tee -a tstdir/file.txt
```

Reply:
Good catch. Yes, sudo is used for the same reason. I will fix it.